### PR TITLE
feat: Add i18n

### PR DESCRIPTION
**Pull request by G3root**
fixes: #14

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -27,4 +27,4 @@ function MyApp({ Component, pageProps }: AppPropsWithLayout) {
   );
 }
 
-export default MyApp;
+export default appWithTranslation<never>(MyApp);
```
Comment (devkiran) [inline pages/_app.tsx:30]: Can we add a proper type here instead of `never`?

Comment (G3root) [reply pages/_app.tsx:30]: I think there is an issue related to that .
https://github.com/i18next/next-i18next/issues/1944

Comment (devkiran) [reply pages/_app.tsx:30]: Noted.

File: locales/en/dashboard.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (devkiran) [inline locales/en/dashboard.json:1]: Let's move translations into common.js. I think we don't want to group the translations into different files.

**Review by devkiran — CHANGES_REQUESTED** [review]
Added some comments.

**Review by devkiran — APPROVED** [review]

**Comment by devkiran** [comment]
@G3root Thanks for the contribution!

**Pull request merged**